src/edit-task.js

```diff
@@ -0,0 +1,4 @@
+export default (arr, index) => {
+  arr[index].description = document.getElementById(`update${index}`).value;
+  return arr;
+};
```

Comment:
- [ ] I am not able to edit the task description, After editing a task if I refresh the page I get the same description as it was before editing. Please make sure editing a task also update the local storage so if a user refreshes the page after editing a task description he should get the edited version of it.

Reply:
@thecodechaser, you can check again by double click on the added description. Edit and click to save. 

![edit function key](https://user-images.githubusercontent.com/86780082/165746867-faa9e250-531d-49ca-ad59-7fb7b80b2589.PNG)